pageload.json

```diff
@@ -0,0 +1,5 @@
+# {"event_id":"e83cab2c681a425bb32d5920d3171c2c","sent_at":"2023-10-25T09:21:51.118Z","sdk":{"name":"sentry.javascript.react","version":"7.75.0"},"trace":{"environment":"prod","release":"frontend@10515a36e3d1f87ebf3a8cc140ef1f071126194d","public_key":"863de587a34a48c4a4ef1a9238fdb0b1","trace_id":"416c8f30220648e08f789bfb1a207087","sample_rate":"1","transaction":"/issues/(searches/:searchId/)","sampled":"true"}}
```

Comment:
Was this intentionally added @JonasBa ?

Reply:
Agh, creeped in from a different PR. Removing it